kipoiseq/dataloaders/sequence.py

```diff
@@ -381,3 +383,52 @@ def get_output_schema(cls):
             output_schema.targets = None
 
         return output_schema
+    
+class AnchoredGTFDl(Dataset):
+    
+    def __init__(self, gtf_file, fasta_file, 
+                 num_upstream, num_downstream,
+                 gtf_filter, anchor_extractor,
```

Comment:
Rename anchor_extractor to `anchor` and allow it to be a string or a function. If string, please hard-code and implemnet anchor/anchor_extractor for a key points on the mRNA such as: TSS, start_codon, stop_codon, polya

Reply:
Done, not 100% sure on the polyA one. It does not seem to be a feature in standard gtf, so I infer it as the gene end.